projects/tcpdump/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2023 ISP RAS
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM sydr/ubuntu20.04-sydr-fuzz
+
+RUN apt-get update && apt-get install -y make cmake flex bison
+
+RUN git clone https://github.com/the-tcpdump-group/libpcap.git libpcap
+RUN cd libpcap && git checkout e0be22909d285fa721379542a754df696c314da3
+
+RUN git clone https://github.com/the-tcpdump-group/tcpdump.git tcpdump
+RUN cd tcpdump && git checkout 1e63a2f6903de9303931f3135db88932d4e5785a
+
+RUN mkdir corpus && mv tcpdump/tests/*.pcap corpus/
+
+COPY build.sh load_sydr_packet.c /
```

Comment:
Rename target (originally it was `fuzz_pretty_print_packet`)

Reply:
You can name source `pretty_print_packet`

And targets for different tools you can name like `pretty_..._sydr`, `pretty_..._fuzz`, etc.

Reply:
done